haystack/nodes/prompt/prompt_node.py

```diff
@@ -569,8 +575,8 @@ def create_invocation_layer(self) -> PromptModelInvocationLayer:
                 )
         raise ValueError(
             f"Model {self.model_name_or_path} is not supported - no invocation layer found."
-            f"Currently supported models are: {self.invocation_layers}"
-            f"Register new invocation layer for {self.model_name_or_path} using the register method."
+            f" Currently supported models are: {self.invocation_layers}"
+            f" Register a new invocation layer for {self.model_name_or_path} using the register method."
```

Comment:
Why the spaces, I am curious.

Reply:
Without spaces, the sentences are mashed together.